mkdocs.yml

```diff
@@ -183,6 +183,7 @@ nav:
       - GPU for services: ./edge/gpus.md
       - gVisor: ./edge/gvisor.md
       - OpenTelemetry: ./edge/open-telemetry.md
+      - Air-gapped installation: ./edge/airgap.md
```

Comment:
Looks like the term is "Air Gap" (rather than air-gap or airgap)

https://en.wikipedia.org/wiki/Air_gap_(networking)

Reply:
Changed this to `Air Gap installation`